src/robusta/core/reporting/callbacks.py

```diff
@@ -27,7 +27,7 @@ def create_for_func(
 
         if not signing_key:
             raise Exception("Cannot create callback request with no signing key. Configure signing_key in globalConfig")
-        action_params = {} if choice.action_params is None else choice.action_params.dict()
+        action_params = {} if choice.action_params is None else choice.action_params.dict(exclude_defaults=True)
```

Comment:
will this break any other callback functionality?

Reply:
no, I checked 
This was actually a bug in the way we handled callback params
Since we weren't using callbacks much, it was un noticed